summaries/Semantically_multi-modal_image_synthesis.md

```diff
@@ -0,0 +1,58 @@
+
+For each semantics, there is a specific controller. By adjusting the controller of a specific class, only the corresponding areas are changed accordingly.
+
+<img src='../images/SMIS_task.png'>
+
+**Notation**
+
+- `M`: semantic segmentation mask
+- `C`: number of semantic classes in the dataset
+- `H`: image height
+- `W`: image width
+- `G`: generator
+- `Z`: latent code
+
+For conducting label-to-image translation,G requires M as conditional input to generate images. However, in order to support multi-modal generation, another input source to control generation diversity. Normally, an encoder is applied to extract Z as the controller. Upon receiving these two inputs, the image output O can be yielded through O = G(Z, M ). However, in the SMIS task, we aim to produce semantically diverse images by perturbing the class-specific latent code which independently controls the diversity of its corresponding class.
+
+For the SMIS task, the key is to divide Z into a series of class-specific latent codes each of which controls only a specific semantic class generation. The traditional convolutional encoder is not an optimal choice because the feature representations of all classes are internally entangled inside the latent code. This phenomenon inspired some architecture modifications in both the encoder and decoder to accomplish the task more effectively.
+
+## GroupDNet 
+
+- The main architecture of GroupDNet takes design inspirations from [SPADE](https://arxiv.org/abs/1903.07291).
+
+- A major modification of GroupDNet is the replacement of typical convolutions to group convolutions to achieve class-specific controllability.
+
+- GroupDNet contains one encoder and one decoder: 
+    - **Encoder**: Let *M<sub>c</sub>* denote the binary mask for class c, *X ∈ R<sup>H×W</sup>* be the input image, then first the operation: *X<sub>c</sub> = M<sub>c</sub> . X*, performs feature disentanglement. The input to the encoder is the concatenation of all the *X<sub>c</sub>* produced. All convolutions inside the encoder have C groups. The encoded Z is comprised of the class-specific Z<sub>c</sub> of all classes which serve as the controllers for their corresponding class *c* in the decoding phase.
+    - **Decoder**: Following the general idea of using all group convolutions in the generator,the convolutional layers in SPADE generator module are replaced with group convolutions resulting in the new module **Conditional Group Normalization (CG-Norm)**. A Conditional Group Block(CG-Block) is made by dynamically merging CG-Norm and group convolutions.
+
+<img src='../images/GroupDNet.png'>
+
+## Main contributions
```

Comment:
Main Contributions

All the headings should have the first letter of the words capitalized, except for words such as 'a', 'an', 'with', etc.